source/js/components/search-form.js

```diff
@@ -0,0 +1,101 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content')
+
+    const url = formEl.dataset.api;
+
+    function init() {
+        formEl.addEventListener('submit', e => {
```

Comment:
[suggestion] Consider passing a named function as your event callback, rather than an anonymous function. It improves readability and also allows for removing event listeners later on, if needed.

Reply:
True that fixed. Thanks.